### PR TITLE
Style nerf

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

Nerfs the armor of greatcoats and stylish jackets, reduces the amount of sunglasses avaliable, adds sunglasses to trade stations.

Style vendor prices and offerings tweaked.

SecHUDs are less stylish.

## Why It's Good For The Game

Style op, pls nerf.

## Changelog
:cl:
tweak: trade stations offer sunglasses
balance: stylish armor nerfed
balance: you can't get sunglasses from loadout unless you're security
balance: secHUDs are no longer as stylish
/:cl:

### CODE REVIEW

File: code/modules/trade/datums/trade_stations_presets/2-uncommon/stylestation.dm [hunk 1]
```diff
@@ -48,6 +49,7 @@
 			/obj/item/clothing/head/feathertrilby,
 			/obj/item/clothing/head/fedora,
 			/obj/item/clothing/head/skull/drip
```
Comment (SirRichardFrancis) [inline code/modules/trade/datums/trade_stations_presets/2-uncommon/stylestation.dm:51]:
```suggestion
			/obj/item/clothing/head/skull/drip,
```

File: code/modules/trade/datums/trade_stations_presets/2-uncommon/stylestation.dm [hunk 2]
```diff
@@ -35,6 +35,7 @@
 			/obj/item/clothing/under/dress/white,
 			/obj/item/clothing/under/helltaker,
 			/obj/item/clothing/under/storage/tracksuit
```
Comment (SirRichardFrancis) [inline code/modules/trade/datums/trade_stations_presets/2-uncommon/stylestation.dm:37]:
```suggestion
			/obj/item/clothing/under/storage/tracksuit,
```

File: code/modules/client/preference_setup/loadout/lists/eyegear.dm [hunk 3]
```diff
@@ -55,15 +55,8 @@
 	display_name = "Sunglasses"
 	path = /obj/item/clothing/glasses/sunglasses
 	cost = 2
+	allowed_roles = list("Ironhammer Operative","Ironhammer Commander","Ironhammer Gunnery Sergeant","Captain","First Officer","Quartermaster","Ironhammer Inspector")
```
Comment (SirRichardFrancis) [inline code/modules/client/preference_setup/loadout/lists/eyegear.dm:58]:
```suggestion
	allowed_roles = list("Ironhammer Operative", "Ironhammer Commander", "Ironhammer Gunnery Sergeant", "Captain", "First Officer", "Quartermaster", "Ironhammer Inspector")
```

**Review by SirRichardFrancis — APPROVED** [review]
Another useless balance change that makes no sense. But code is okay.

**Comment by FortuneMaster** [comment]
great pr, style should be nerfed😎

File: code/modules/clothing/glasses/hud.dm [hunk 4]
```diff
@@ -71,6 +72,7 @@
 	name = "Ironhammer tactical HUD"
 	desc = "Flash-resistant goggles with inbuilt combat and security information."
 	icon_state = "swatgoggles"
+	style = STYLE_LOW
```
Comment (DiaFRAME444) [inline code/modules/clothing/glasses/hud.dm:75]: Is this needed?
Inspector is supposed to be a Style user, this hits him too.
And the rest of ironhammer clothes are also low Style already so this won't even matter.

(Accidentally posted this as a review the first time around, ignore that)

Comment (Humonitarian) [reply code/modules/clothing/glasses/hud.dm:75]: He has access to sunglasses in the inspector office, and can take more through loadout.

Comment (Humonitarian) [reply code/modules/clothing/glasses/hud.dm:75]: Tho honestly sechud lenses would be cool

**Comment by Humonitarian** [comment]
Outdated changes

**Pull request closed without merging**